src/components/markdown-renderer/replace-components/mermaid/mermaid-chart.tsx

```diff
@@ -66,7 +67,7 @@ export const MermaidChart: React.FC<MermaidChartProps> = ({ code }) => {
         showError(message || t('renderer.mermaid.unknownError'))
       }
     })
-                     .catch(() => showError('Error while loading mermaid'))
+                                                      .catch(() => showError('Error while loading mermaid'))
```

Comment:
Same as my comment above regarding the then-catch intendation.

Reply:
fixed